WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemUiStateHelper.kt

```diff
@@ -282,9 +287,10 @@ class PostListItemUiStateHelper @Inject constructor(private val appPrefsWrapper:
         isLocalDraft: Boolean,
         isLocallyChanged: Boolean,
         uploadUiState: PostUploadUiState,
-        hasUnhandledConflicts: Boolean
+        hasUnhandledConflicts: Boolean,
+        hasAutoSave: Boolean
     ): Int? {
-        val isError = uploadUiState is PostUploadUiState.UploadFailed || hasUnhandledConflicts
+        val isError = uploadUiState is PostUploadUiState.UploadFailed || hasUnhandledConflicts || hasAutoSave
```

Comment:
I personally wouldn't consider this as an error - the red label feels weird. But I'll leave that decision on @osullivanchris